assets/js/modules/adsense/index.js

```diff
@@ -19,22 +19,19 @@
 /**
  * WordPress dependencies
  */
-import { addFilter } from '@wordpress/hooks';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import { AREA_DASHBOARD_EARNINGS } from '../../googlesitekit/widgets/default-areas';
-import { fillFilterWithComponent } from '../../util';
```

Comment:
After removing this, the `fillFilterWithComponent` function is not needed anymore and we can get rid of it too. Please, remove it too.

Reply:
https://github.com/google/site-kit-wp/pull/3823/commits/4a48393b2f9c98c5dc3adb63e35e5d6b13f5b081